test/mock-rippled.ts

```diff
@@ -119,12 +119,26 @@ export function createMockRippled(port) {
   mock.on('request_config', function(request, conn) {
     assert.strictEqual(request.command, 'config')
     conn.config = _.assign(conn.config, request.data)
+    conn.send(
```

Comment:
a response is expected for every request, otherwise the Connection rejects with a timeout error.